### DISCUSSION TITLE
How to distinguish results from different phases in html report

### DISCUSSION THREAD

**Discussion by Homer213** [artilleryio/artillery#1590] [Artillery]
Hi,

I have 3 phases in my script and I would like to know if there is a way on how to check the results of each phases in the html report. Thank you in advance!

phases:
    - duration: 30
      arrivalRate: 1
      name: "Warm up"
    - duration: 30
      arrivalRate: 1
      rampTo: 20
      name: "Ramp up"
    - duration: 300
      arrivalRate: 20
      maxVusers: 50
      name: "Sustain load"

**Comment by hassy** [comment]
hi @Homer213 👋 could you explain what you mean by "results of each phase"? something like the final summary report, but for each phase instead?

**Reply by Homer213** [reply]
hi @hassy ... thank you for your response.. regarding your question, yes what i mean is in the final summary in the html report if we can have sections for each phase.. thanks.
